taxonomies/categories.txt

```diff
@@ -27513,9 +27521,6 @@ ciqual_food_code:en:19666
 ciqual_food_name:en:Petit-Suisse like fresh cream cheese, plain, around 9% fat
 ciqual_food_name:fr:Fromage frais type petit suisse, nature, 10% MG environ
 
-
-en:Coffee creamers
```

Comment:
I added that for the US import. Not sure what parent to add. Dairy products ?

Reply:
From what I see Coffee creamers does not seem to contain milk. We already have a category "Creamers" in milk subtitute, I assume it's the same thing

Reply:
@moon-rabbitOFF @teolemon  It is the same thing indeed, let's just add coffee creamers as a synonym of creamers.